### DISCUSSION TITLE
Genome Size and VCF2RADdata

### DISCUSSION THREAD

**Discussion by andrea-bellucci** [lvclark/polyRAD#35] [General]
Hello. I was starting to play around and trying to import my vcf file. I get this error

> Indexing VCF.
> [E::hts_idx_check_range] Region 536950282..536950283 cannot be stored in a tbi index. Try using a csi index
> Error in value[[3L]](cond) : index build failed

I am assuming it is because chromosome size is too big for tabix. Checking VCF2RADdata it can be seen that calls Rsamtools::TabixFile for vcf indexing which i assume it calls tabix eventually. But there s no option in Rsamtools::TabixFile to change index tipe and having a csi index in the same folder would help because  VCF2RADdata  check for .tbi index presence. 
So i am a bit stuck here. Any idea how to get around this without splitting all chromosomes in the reference? 
Thanks

**Comment by lvclark** [comment]
This is something I can definitely fix in the code.  I know that
VariantAnnotation can read csi indices, at least.

**Reply by lvclark** [reply]
Ok, I just pushed a fix to GitHub, and it should propagate to R-universe within a few hours.  If you bgzip and csi-index your VCF before running VCF2RADdata, it should work.  Let me know if you run into issues.

**Reply by andrea-bellucci** [reply]
Thanks a lot for the help. I will try it out asap.

**Reply by andrea-bellucci** [reply]
Works great now providing vcf.bgz file and csi index. Thanks again for the help!
